### PR TITLE
#799 Listbox-button: support fluid version

### PR DESCRIPTION
**Pull request by partynuts**
## Description
support listbox-btn being fluid

## References
https://github.com/eBay/ebayui-core/issues/799
http://localhost:6006/?path=/story/ebay-listbox-button--fluid

### CODE REVIEW

**Comment by ianmcburnie** [comment]
@seangates @partynuts Sorry, but this needs to go into 4.4.0 branch, which hadn't been created. I'm going to change the target but it might mean you now need to rebase your feature branch with 4.4.0 branch.

**Comment by ianmcburnie** [comment]
This will need a change on the skin side in 9.4.0:

```css
.listbox-button--fluid {
  width: 100%;
}
```

Then you can reference that new class.

**Comment by ianmcburnie** [comment]
@seangates Let us know when you have this one in shape for review.

**Comment by seangates** [comment]
@ianmcburnie Should be good now! 👍

File: .gitignore [hunk 1]
```diff
@@ -12,6 +12,7 @@ dist
 .browser-refresh
 integration/template.marko
 integration/browser.json
+.idea
```
Comment (ianmcburnie) [inline .gitignore]: Curious, what file extension is this? And do we really need it?

File: src/components/ebay-listbox-button/README.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ Name | Type | Stateful | Required | Description
 `name` | String | No | Yes | used for the `name` attribute of the native `<select>`
 `selected` | Number | Yes | n/a | allows you to set the selected index option to `selected`
 `borderless` | Boolean | No | No | whether button has borders
+`fluid` | Boolean | No | No | wether listbox width is 100%
```
Comment (ianmcburnie) [inline src/components/ebay-listbox-button/README.md]: Minor, but a typo on 'wether' (unless it's some americanism I'm unfamiliar with).

File: .gitignore (hunk 1 above)
Comment (seangates) [reply .gitignore]: This is for IntelliJ-type editors. There's actually a lot of folks within eBay using it. It's not the first time I've seen it added (I simply moved it to be next to the one for VSCode).

Not "necessary", but is nice to have.

File: src/components/ebay-listbox-button/README.md (hunk 2 above)
Comment (seangates) [reply src/components/ebay-listbox-button/README.md]: No, that's 100% a typo.

**Comment by seangates** [comment]
@ianmcburnie Fixed the typo.

**Pull request merged**